src/integration-test/kotlin/integration/helper/IntegrationHelperUtil.kt

```diff
@@ -362,6 +476,19 @@ class IntegrationHelperUtil {
             .get()
     }
 
+    /**
+     * Send HTTP POST request to registration service to register user
+     * @param name - user name
+     * @param pubkey - user public key
+     */
+    fun sendRegistrationRequest(name: String, pubkey: PublicKey): khttp.responses.Response {
```

Comment:
can we create public key right here?

Reply:
I think it is responsibility of caller, so she/he can send Iroha txs.